extension-config.json

```diff
@@ -13,6 +13,14 @@
       ],
       "type": "portals"
     },
+    {
+      "id": "ProductPropertiesBasePdp",
+      "path": "frontend/portals/ProductPropertiesBasePdp",
+      "target": [
+        "product.properties"
```

Comment:
not sure about this since it will "burn" this portal for other extension. So they can't be used together with this extension anymore, even the feature is not used